### DISCUSSION TITLE
Q: 后台无法显示“待审核”评论 || Q: The "pending review" comment cannot be displayed in the background

### DISCUSSION THREAD

**Discussion by HoronLee** [walinejs/waline#2664] [Q&A] [answered]
### Describe the bug | 问题描述

waline 版本：当前最新
在 waline 管理后台选择*待审核*评论无法得到评论列表，报错*500 Internal Server Error*，在 vercel 的 log 中，报错为
```
Aug 13 12:19:27.34
GET
500
waline.horon.top
/api/comment
Error: Input data should be a String at MarkdownIt.parse (/var/task/node_modules/markdown-it/dist/index.cjs.js:5488:11) at MarkdownIt.render (/var/task/node_modules/markdown-it/dist/index.cjs.js:5508:36) at /var/task/node_modules/@waline/vercel/src/service/markdown/index.js:61:43 at formatCmt (/var/task/node_modules/@waline/vercel/src/controller/comment.js:56:21) at async Promise.all (index 0) at module.exports.getAdminCommentList (/var/task/node_modules/@waline/vercel/src/controller/comment.js:653:13) at module.exports.getAction (/var/task/node_modules/@waline/vercel/src/controller/comment.js:90:18) at cors (/var/task/node_modules/@koa/cors/index.js:109:16) at /var/task/node_modules/@waline/vercel/src/middleware/version.js:5:3 at /var/task/node_modules/@waline/vercel/src/middleware/prefix-warning.js:27:3
```
<img width="1501" alt="a32b1101-d2a0-4f3d-966b-0025e492e267" src="https://github.com/user-attachments/assets/985e3260-e990-4377-9792-de8e213c3303">
<img width="1492" alt="e236d6b7-5eca-438b-b404-34afadc68648" src="https://github.com/user-attachments/assets/45f33699-4ccf-4f9e-ae6e-1cb5f5c8ae48">

### Website URL | 问题网站

https://waline.horon.top

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

waline version: the latest
Select *pending review* comments in the waline management background. The comment list cannot be obtained, and the error *500 Internal Server Error* is reported. In the log of vercel, the error is reported as
```
Aug 13 12:19:27.34
GET
500
waline.horon.top
/api/comment
Error: Input data should be a String at MarkdownIt.parse (/var/task/node_modules/markdown-it/dist/index.cjs.js:5488:11) at MarkdownIt.render (/var/task/node_modules/markdown- it/dist/index.cjs.js:5508:36) at /var/task/node_modules/@waline/vercel/src/service/markdown/index.js:61:43 at formatCmt (/var/task/node_modules/ @waline/vercel/src/controller/comment.js:56:21) at async Promise.all (index 0) at module.exports.getAdminCommentList (/var/task/node_modules/@waline/vercel/src/controller/comment .js:653:13) at module.exports.getAction (/var/task/node_modules/@waline/vercel/src/controller/comment.js:90:18) at cors (/var/task/node_modules/@koa /cors/index.js:109:16) at /var/task/node_modules/@waline/vercel/src/middleware/version.js:5:3 at /var/task/node_modules/@waline/vercel/src/ middleware/prefix-warning.js:27:3
```
<img width="1501" alt="a32b1101-d2a0-4f3d-966b-0025e492e267" src="https://github.com/user-attachments/assets/985e3260-e990-4377-9792-de8e213c3303">
<img width="1492" alt="e236d6b7-5eca-438b-b404-34afadc68648" src="https://github.com/user-attachments/assets/45f33699-4ccf-4f9e-ae6e-1cb5f5c8ae48">

### Website URL | Problem website

https://waline.horon.top

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
能去数据库里看下这条评论的内容么，看起来内容 Markdown 渲染失败嘞

---
Can you check the content of this comment in the database? It seems that the Markdown rendering of the content failed

**Reply by HoronLee** [reply]
> 能去数据库里看下这条评论的内容么，看起来内容 Markdown 渲染失败嘞
> 
> Can you check the content of this comment in the database? It seems that the Markdown rendering of the content failed

我查看了数据库中的那条记录，确实有问题，我删除了那条记录后待审核中就没有任何内容了，应该就是渲染失败了，感谢提醒！
